### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akshara-sun**
Added a few links + created a new section containing practice tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +138,9 @@
 * [github.com/amilajack/PitchDetect](https://github.com/amilajack/PitchDetect)
 * [webaudiodemos.appspot.com/input](https://webaudiodemos.appspot.com/input/)
 * [Audio looper](https://github.com/cwilso/Looper.js)
+* [chord progressions](https://arp.jake.fun)
+
```
Comment (amilajack) [inline README.md]: Added two extra new lines here

**Review by amilajack — CHANGES_REQUESTED** [review]
Overall, LGTM!

File: README.md [hunk 2]
```diff
@@ -199,5 +206,7 @@
 * [Subtractive synths explained](https://www.residentadvisor.net/features/1351)
 
 ## Experiments
-
 * [generative.fm](https://generative.fm): Endlessly unique ambient music
+
+## Practice Tools
```
Comment (amilajack) [inline README.md]: I think we should move this to an existing section. Post people seeing this might not know what "Practice Tools" is.

**Review by amilajack — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -39,6 +39,7 @@
 * [github.com/kylestetz/AudioKeys](https://github.com/kylestetz/AudioKeys)
 * [slang.kylestetz.com](http://slang.kylestetz.com/)
 * [kylestetz.github.io/Sentencer](http://kylestetz.github.io/Sentencer/)
+* [Ukelele Practice](https://ukulele-practice.glitch.me)
```
Comment (amilajack) [inline README.md]: What do you think about adding this to the "live experiments" section?

**Review by amilajack — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (akshara-sun) [reply README.md]: Do you mean "live demos" or do you wanna make a separate section all together?

Comment (amilajack) [reply README.md]: 👍

**Review by amilajack — APPROVED** [review]

**Pull request merged**